packages/archivematica_cleanup/src/index.ts

```diff
@@ -0,0 +1,51 @@
+import * as Sentry from "@sentry/node";
+import "./env";
+
+const env = process.env["ENV"] ?? "";
+Sentry.init({
+  dsn: process.env["SENTRY_DSN"] ?? "",
+  tracesSampleRate: 1.0,
+  environment: env === "local" ? `local-${process.env["DEV_NAME"] ?? ""}` : env,
+});
+
+export const cleanupDashboard = async (): Promise<void> => {
+  try {
+    const transferDeleteResponse = await fetch(
+      `${process.env["ARCHIVEMATICA_BASE_URL"] ?? ""}/api/transfer/delete`,
```

Comment:
I don't see this in [their API docs](https://www.archivematica.org/en/docs/archivematica-1.15/dev-manual/api/api-reference-archivematica/#api-reference-archivematica) - I only see deleting one at a time. Do you know where that's documented? (I just searched "delete" in the page so could easily have missed it.)